bitbot/links_test.go

```diff
@@ -60,3 +60,19 @@ func TestGetHTMLTitleWithEmptyTitle(t *testing.T) {
 		t.Fail()
 	}
 }
+
+func TestAntiUrlFlood(t *testing.T) {
+	t.Log("Note : If the previous test failed, this one will hang indefinitely")
+
+	message := "https://secops.space/"
```

Comment:
Test shouldn't rely on external infrastructure to work.  Might want to mock a small http server here and call a lookup on localhost.